domains/transfer/Transfer.API/API/Transfer/Api/Controllers/TransferAgreementsController.cs

```diff
@@ -27,11 +27,10 @@ namespace API.Transfer.Api.Controllers;
 [ApiVersion(ApiVersions.Version20240103)]
 [Route("api/transfer/transfer-agreements")]
 public class TransferAgreementsController(
-    ITransferAgreementRepository transferAgreementRepository,
     IProjectOriginWalletService projectOriginWalletService,
     IHttpContextAccessor httpContextAccessor,
     ITransferAgreementProposalRepository transferAgreementProposalRepository,
```

Comment:
Shouldnt this also be deleted?

Reply:
Yes. Needs to be enforced throughout the solution as well